osm/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/osm/ProcessingChainOSMTest.groovy

```diff
@@ -552,29 +552,24 @@ class ProcessingChainOSMTest extends ChainProcessAbstractTest {
                 "geoclimatedb" : [
                         "folder" : "${dirFile.absolutePath}",
                         "name" : "geoclimate_chain_db;AUTO_SERVER=TRUE",
-                        "delete" :true
+                        "delete" :false
                 ],
                 "input" : [
-                        "osm" : ["Pont-de-Veyle"]],
+                        "osm" : ["Nantes"]],
```

Comment:
did you try your version on cities that had the problem of intersection before we added the precision_reducer ?

Reply:
You are right